people/outiedorfling.md

```diff
@@ -0,0 +1,35 @@
+# Outie Dorfling
+
+
+## Personal blurb
+I really like helping people break down their workflow to help figure out how to simplify it.
+I'm a ham radio operater and enjoy taking electronics apart with the intent to maybe one day reasemble them.
+
+## Skill Set
+I have experience with these languages/frameworks/systems, I really enjoy automating a lot of the workflow, freeing up people to focus on more important work
```

Comment:
Might be worth adding your own text here :D 
`these languages/frameworks/systems`